tests/test_cli.py

```diff
@@ -18,6 +18,8 @@
 )
 from chaoslib.settings import CHAOSTOOLKIT_CONFIG_PATH
 from click.testing import CliRunner
+import tempfile
+
```

Comment:
The `log_file` is already created for you in [`tests/conftest.py`](https://github.com/chaostoolkit/chaostoolkit/blob/master/tests/conftest.py) so you don't need to recreate it

```suggestion
```